newsroom/companies/views.py

```diff
@@ -19,61 +19,44 @@ def index():
         companies=companies)
 
 
-@blueprint.route('/companies/new', methods=['GET', 'POST'])
+@blueprint.route('/companies/search', methods=['GET'])
+@admin_only
+def search():
+    companies = list(query_resource('companies', max_results=50))
+    response = flask.current_app.response_class(
+        response=json.dumps(companies, default=json_serialize_datetime_objectId),
```

Comment:
it should be possible to use [flask.jsonify](http://flask.pocoo.org/docs/0.12/api/#flask.json.jsonify) for this, app json encoder is configured now, you might need to rebase to get it working